hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -1077,6 +1077,7 @@ private GetFileStatusResponse getOzoneFileStatus(
         .setVolumeName(keyArgs.getVolumeName())
         .setBucketName(keyArgs.getBucketName())
         .setKeyName(keyArgs.getKeyName())
+        .setRefreshPipeline(keyArgs.getRefreshPipeline())
```

Comment:
Do we need to set refreshPipeline to true in list status also?

Reply:
@bharatviswa504 Yes, FS listStatus API is invoked by MapReduce jobs to get the located file blocks.

Reply:
So is that missing in OzoneManagerRequestHandler.java?

Reply:
Yes, it is not being set to true in OzoneManagerRequestHandler#listStatus currently.